java/java.lsp.server/vscode/package.json

```diff
@@ -108,6 +108,11 @@
 								],
 								"description": "The specified console to launch the program.",
 								"default": "internalConsole"
+							},
+							"args": {
+                                "type": "string",
```

Comment:
Space/tab issue? The indentation doesn't look right.

Reply:
Fixed in ef8b981300